docs/grammar.md

```diff
@@ -297,12 +315,24 @@ Note that `%ignore` directives cannot be imported. Imported rules will abide by
 
 Declare a terminal without defining it. Useful for plugins.
 
+**Syntax:**
+```html
+%declare <TERMINAL>
+%declare <rule>
+```
+
 ### %override
 
 Override a rule or terminals, affecting all references to it, even in imported grammars.
 
 Useful for implementing an inheritance pattern when importing grammars.
 
+**Syntax:**
+```html
+%override <TERMINAL> ... terminal definition ...
```

Comment:
Not accurate..

Reply:
https://github.com/lark-parser/lark/blob/master/lark/load_grammar.py#L162-L165 and https://github.com/lark-parser/lark/blob/master/lark/load_grammar.py#L1260-L1262 make it look like you can override a terminal, by supplying the full definition, preceded by `%override`.  What did I misunderstand?

Again, if the description was less words and more productions, I'd have just written `<override> ::= "%override" (rule|term)`.

Reply:
Sorry, I should have been more specific.

The syntax is `%override <terminal definition>`, or `%override <TERMINAL> : <items>`

Either way, but the way you write it is confusing, and would make the reader think the syntax is different.